python/ql/src/Functions/ModificationOfParameterWithDefault.ql

```diff
@@ -19,5 +19,5 @@ from
   ModificationOfParameterWithDefault::Configuration config, DataFlow::PathNode source,
   DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ flows to here and is mutated.", source.getNode(),
-  "Default value"
+select sink.getNode(), source, sink, "This value modification depends on a $@.", source.getNode(),
+  "default value"
```

Comment:
This one sounds a bit strange to me ("'value modification'? What's that?"), but I couldn't come up with a better phrasing.

Also, "depends on a default value" seems too weak of a statement to me. It not only _depends_ on a default value, it actively _mutates_ said default value. 

Edit: Perhaps "`This expression mutates a default value.`" would be better?

Reply:
Actually, `s/mutates/modifies/` might be even better.

Reply:
> Also, "depends on a default value" seems too weak of a statement to me. It not only _depends_ on a default value, it actively _mutates_ said default value.

You're right, it's a `DataFlow::Configuration` and not a `TaintTracking::Configuration`, so _depends_ is too weak a statement.   
"This expression mutates a default value" sounds good to me 👍